services/mac_manager/pom.xml

```diff
@@ -88,11 +88,7 @@
             <groupId>io.springfox</groupId>
             <artifactId>springfox-swagger2</artifactId>
             <version>2.6.1</version>
-        </dependency>
-        <dependency>
-            <groupId>io.springfox</groupId>
-            <artifactId>springfox-swagger-ui</artifactId>
-            <version>2.2.2</version>
```

Comment:
Let us keep io.springfox as they are used to generate Swagger doc/UI

Reply:
Yes, I will keep io.springfox